### ISSUE TITLE
Get MoBa person on board

### ISSUE THREAD

**Issue by troettge** [troettge/Research-NORRN#21]

**Comment by troettge** [comment]
Find a collaborator who has experience with the [MoBa](https://www.fhi.no/en/studies/moba/) data set. Ideally an open science skeptic?

**Comment by troettge** [comment]
tagging @julienmayor here as you might have leads on MoBa folks

**Comment by julienmayor** [comment]
Yes - a good person could be Ludvig (https://www.sv.uio.no/psi/english/people/aca/ludvigdb/index.html). Alternatively, Tamara mentioned Jasmina (https://www.sv.uio.no/psi/english/people/aca/jasminbu/index.html). I can approach both if you want me to.

**Comment by Tamara-Kalan** [comment]
Maybe start with Ludvig?

**Comment by troettge** [comment]
@julienmayor would you be willing to ask Ludvig if he wants to be involved?

**Comment by julienmayor** [comment]
Email sent - I'll keep you posted once I hear from him.

**Comment by Tamara-Kalan** [comment]
Looking forward to hearing how the meeting goes :-)

**Comment by julienmayor** [comment]
Very nice meeting. Main points here:
- Indeed, we need to frame the project VERY carefully so that we highlight challenges and solutions/opportunities in related to epidemiology, rather that on QRP.
- Ludvig suggested we (or he) get in touch with Laurie Hannigan (https://scholar.google.co.uk/citations?user=WvCEWZgAAAAJ&hl=en, U. Bristol - previously working with MoBa) who developed an R package (https://github.com/psychgen/phenotools  to adopt systematic data cleaning practice for MoBa data, etc)
- Adrian Askelund (working with Laurie) would also be a good fit.
- Some young researchers upload their scripts to github (e.g., Espen Eilertsen, Rosa Cheesman) or have favourable views on Open Science. We may want to also approach some of them.
- Apparently it is a requirement for researchers using the MoBa to upload their script so that MoBa stores them (internal control??)
- However, apparently MoBa guidelines explicitly forbid researchers to share data (even aggregated)
- Finally, after 20 years (?) there is a new head of the MoBa with Ragnhild Brandlistuen (since last year, I believe) - we could get in touch with her to see if there are any OS-related changes in the pipeline?

**Comment by Tamara-Kalan** [comment]
Thanks for the updates, Julien! Let me know how I can contribute. We probably need to discuss how to proceed

**Comment by troettge** [comment]
Thank you, Julien, for your update. This is super insightful. I know Rosa Cheesman, so I can ask her if she wants to be involved. I will also write an email to Laurie Hannigan. Did Ludwig articulate interest in being on board? Would you, Julien, be willing to reach out to Ragnhild Brandlistuen to probe if there are changes upcoming? Maybe they are interested in being involved which would make this much easier?

**Comment by troettge** [comment]
UPDATE: Sent out emails to Lauri, Adrian and Rosa.

**Comment by julienmayor** [comment]
Great to see positive responses. Yes, I think that providing that we are being constructive in the project, Ludwig is interested in being involved. 
Cf Ragnhild, I'm due to talk to her at the end of April, actually to potentially launch a MoBa analysis - that, of course, would be preregistered :o) - would that be alright if I talk to her then? It may be easier to probe the water "live"...
